app/services/helper_functions.py

```diff
@@ -0,0 +1,36 @@
+from app.services.geolocalization_services.user_localization_and_distance import (
+    get_coordinates,
+)
+from app.services.robot_match.robot_selector import to_shape, wkt_to_tuple
+
+
+def robot_to_dict(table):

```

Comment:
i guess that it takes a list of robots and return a list of dictorianries. Maybe it will be better to just has a `.serialize` method in the `Base` class that will go through the fields. I guess it may be already implement by sqlalchemy but has a slightly different name